src/Libraries/Microsoft.Extensions.AI/ChatCompletion/CachingChatClient.cs

```diff
@@ -44,7 +44,7 @@ protected CachingChatClient(IChatClient innerClient)
     /// </remarks>
     public bool CoalesceStreamingUpdates { get; set; } = true;
 
-    /// <inheritdoc />
+    /// <inheritdoc cref="IChatClient.CompleteAsync(IList{ChatMessage}, ChatOptions?, CancellationToken)"/>
```

Comment:
@gewarren what's the reason for needing the explicit cref?

Reply:
@RussKie [learn.microsoft.com doesn't understand inheritdoc from something that itself has inheritdoc](https://buildapi.docs.microsoft.com/Output/PullRequest/500c749d-de78-3d4a-00e4-269d599bf9ec/202501050810051900-10808/BuildReport?accessString=8150976b02bdfaba2a8d4273cf5fc53d484938991b2766fda7369ec3e0c63520). If you don't like this change, I can try to get the Learn platform team to fix this instead.

Reply:
At least I assume it's not working on Learn because the parent type also has `inheritdoc` on this member. I'm not 100% sure it's the problem, but it's an educated guess.

Reply:
It'd be great if the platform could fix it. Otherwise given there's compiler / VS support, we'll likely continue to have a long tail of such issues reoccurring (in this and other repos).

Reply:
https://dev.azure.com/ceapex/Engineering/_workitems/edit/1026690